api/server/sdk/cloud_backup.go

```diff
@@ -86,6 +86,67 @@ func (s *CloudBackupServer) Create(
 	}, nil
 }
 
+// GroupCreate creates a backup for a list of volume or group
+func (s *CloudBackupServer) GroupCreate(
+	ctx context.Context,
+	req *api.SdkCloudBackupGroupCreateRequest,
+) (*api.SdkCloudBackupGroupCreateResponse, error) {
+
+	if s.driver(ctx) == nil {
+		return nil, status.Error(codes.Unavailable, "Resource has not been initialized")
+	}
+	credId := req.GetCredentialId()
+	var err error
+	if len(req.GetGroupId()) == 0 && len(req.GetVolumeIds()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply a group ID or list of volume IDs")
+	}
+	if len(req.GetCredentialId()) == 0 {
+		credId, err = s.defaultCloudBackupCreds(ctx)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// Check ownership for all volume IDs
+	if len(req.GetVolumeIds()) > 0 {
+		if err := checkAccessFromDriverForVolumeIds(ctx, s.driver(ctx), req.GetVolumeIds(), api.Ownership_Read); err != nil {
+			return nil, err
+		}
+	}
+
+	// Check ownership for group ID
+	if len(req.GetGroupId()) > 0 {
+		if err := checkAccessFromDriverForGroupId(ctx, s.driver(ctx), req.GetGroupId(), api.Ownership_Read); err != nil {
+			return nil, err
+		}
+	}
+
```

Comment:
If both groupID and volume list is not specified and only labels are provided, permissions perhaps should be checked for those volumes with matching labels?